srcpkgs/teamspeak3/template

```diff
@@ -13,12 +13,12 @@ license="Proprietary"
 homepage="http://www.teamspeak.com/"
 if [ "$XBPS_TARGET_MACHINE" = "x86_64" ]; then
 	_pkg="TeamSpeak3-Client-linux_amd64-${version}"
-	distfiles="http://teamspeak.gameserver.gamed.de/ts3/releases/${version}/${_pkg}.run"
-	checksum=53fda7bc9137c712f1d30163b54981e4ae9c46bec76585a9b0fa87397dd19859
+	distfiles="http://dl.4players.de/ts/releases/${version}/${_pkg}.run"
+	checksum=63f3781c1f49ccc26368d9238a3e2aeca0e57dcb65a1c4022341ad7e8ba4fe9d
 elif [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
 	_pkg="TeamSpeak3-Client-linux_x86-${version}"
-	distfiles="http://teamspeak.gameserver.gamed.de/ts3/releases/${version}/${_pkg}.run"
-	checksum=222e8abb24de9e3ea00fca10be32340ad88859a4d811afa644c5096aada4996d
+	distfiles="http://dl.4players.de/ts/releases/${version}/${_pkg}.run"
+	checksum=53e83aea770c53d3a513b50d4f041915d3ac1ec997638505c87c1c96a71f76ae
 else
 	broken="No known upstream client for this architecture"
```

Comment:
Why not `only_for_archs="i686 x86_64"` ?

Reply:
I don't know, I didn't make this template and I'm new to editing xbps packages. 

I'll add it in.

Reply:
IMO it was fine as it was. `broken=` is mostly the preferred way since it can give you additional information which `only_for_archs` can't.

Reply:
`only_for_archs` because the [downloads page](https://www.teamspeak.com/en/downloads) provides only those...

Reply:
...but it's okay, no problem. :+1: